src/SFA.DAS.EmployerFinance.Web/Views/Shared/_ZenDeskApiPush.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿<script type="text/javascript">
+    zE(function() {
+        zE.identify({
+            name: '@ViewBag.GaData.UserName',
```

Comment:
Might be worth adding a separate filter to populate ViewBag.ZendeskApiData. Because of the viewbag being dynamic if someone decided to rip out all GA then they would remove the GAData and not know they inadvertently affected the zendesk api push

Reply:
Done